pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -223,7 +223,7 @@ func (r *ReconcileAccountClaim) Reconcile(request reconcile.Request) (reconcile.
 				// Update the status on AccountClaim
 				return reconcile.Result{}, r.statusUpdate(reqLogger, accountClaim)
 			}
-			waitMsg := fmt.Sprintf("%s is not Ready yet requeuing in %d seconds", byocAccount.Name, waitPeriod)
+			waitMsg := fmt.Sprintf("%s is not Ready yet (current status: %s) requeuing in %d seconds", byocAccount.Name, byocAccount.Status.State, waitPeriod)
 			reqLogger.Info(waitMsg)
```

Comment:
Could we add it this way instead:

```suggestion
			reqLogger.Info(waitMsg, "Account Status", byocAccount.Status.State)
```

Reply:
LGTM, @rogbas suggested the current format for the log message, if he 👍🏻 then I will commit your suggestion